FYI: set dwz_file in dwarf2read.c
http://sourceware.org/ml/gdb-patches/2012-12/msg00354.html
http://sourceware.org/ml/gdb-cvs/2012-12/msg00058.html

### src/gdb/ChangeLog	2012/12/11 18:45:23	1.14904
### src/gdb/ChangeLog	2012/12/11 19:50:52	1.14905
## -1,5 +1,9 @@
 2012-12-11  Tom Tromey  <tromey@redhat.com>
 
+	* dwarf2read.c (dwarf2_get_dwz_file): Set 'dwz_file'.
+
+2012-12-11  Tom Tromey  <tromey@redhat.com>
+
 	* dsrec.c (report_transfer_performance): Don't declare.
 	(load_srec): Use gettimeofday, print_transfer_performance.
 	* symfile.c (report_transfer_performance): Remove.
--- src/gdb/dwarf2read.c	2012/12/10 22:02:34	1.720
+++ src/gdb/dwarf2read.c	2012/12/11 19:50:53	1.721
@@ -2112,6 +2112,7 @@
 
   do_cleanups (cleanup);
 
+  dwarf2_per_objfile->dwz_file = result;
   return result;
 }
 
